### DISCUSSION TITLE
使用es catalog，用esquery函数时报错

### DISCUSSION THREAD

**Discussion by Jun-yuxi** [apache/doris#20043] [A - General / Q&A]
执行sql为：
select * from idl_pub_lawsuit_case_dd 
where esquery(content_html,'{
    "term": {
      "content_html": "阿里"
    }
}') ;
报错为：
errCode = 2, detailMessage = (11.159.185.134)[NOT_IMPLEMENTED_ERROR]Fake function fake do not support execute

**Comment by LemonLiTree** [comment]
which version?
